src/types/domain/Player.ts

```diff
@@ -3,6 +3,7 @@ import { Match } from "./Match";
 export type Player = {
     name: string;
     matches: Match[];
+    validMatches: number; // Games we want to take into account when calculating stats (usually 4 player pod)
```

Comment:
i'm going to be super nitpicky about changes in types folder because they are foundational to the app :) 

Can we put this in a /** **/ comment block above "validMatches" (see below for an example of how to properly JS Doc so intellissense picks it up)

Reply:
Also, given that this is a number, can we name this "validMatchCount"

Reply:
Finally, the comment itself is too vague. Maybe something like: 
"The number of matches considered valid for the purposes of calculating stats"

namely, these are not "games". this a COUNT of games. But I also don't want to use the concept of a "Game" here. The app only has concept of "Matches".

Reply:
Fixed.